events/lastAvatarStandingTrivia/clientScripts/triviaMasterClientStandard.js

```diff
@@ -779,184 +786,213 @@
 
     function onWebEventReceived(event) {
         if (typeof event === 'string') {
-            event = JSON.parse(event);
+            try {
+                event = JSON.parse(event);
+            } catch (e) {
+                console.log("error parsing web event", e);
+                return;
+            }
             if (event.app === 'trivia') {
-                switch (event.type) {
-                    case 'catalog':
-                        switch (event.value) {
-                            case "Default Catalog":
-                                useGoogle = false;
-                                break;
-                            case "Custom Catalog":
-                                useGoogle = true;
-                                triviaURL = SECRETS.trivia_URL;
-                                answer = Window.prompt("What is the tab name of the sheet to use?", "");
-                                if (answer === null) {
-                                    tablet.emitScriptEvent("nothingEntered");
-                                    useGoogle = false;
-                                } else {
-                                    gSheetTabName = answer;
+                var newTime = new Date();
+                if (newTime - lastTime > 100) {
```

Comment:
what does this do?

Reply:
I was seeing multiple emitWebEvent signals from the HTML, this was my attempt to protect the app js listener from triggering events in multiples.